README.md

```diff
@@ -4,3 +4,9 @@ project based on this template:
 ```
 npx degit graph-paper-org/app-template graph-paper-app
 ```
+
+## Development
+
+Your new project will include [developer documentation](docs/development.md) but
+not this README. You may want to create your own README file which links to the
+developer documentation.
```

Comment:
The developer documentation could live in a README, but this README file wouldn't be especially useful to teams that are *using* this template and degit doesn't provide a way to provide an alternative README file.